src/goto-programs/goto_contractor.cpp

```diff
@@ -221,6 +231,25 @@ goto_contractort::create_constraint_from_expr2t(irep_container<expr2t> expr)
   return c;
 }
 
+bool goto_contractort::is_unsupported_operator(irep_container<expr2t> expr)
```

Comment:
you can rewrite this as `return     is_arith_expr(e) ||
    is_constant_number(e) ||
    is_symbol2t(e) ||
    is_notequal2t(e) ||
    is_equality2t(e) ||
    is_not2t(e) ||
    is_modulus2t(e) ||
    is_or2t(e) ||
    is_and2t(e));`

Reply:
Done